classes/class-sensei-media-attachments.php

```diff
@@ -180,7 +180,7 @@ public function display_attached_media() {
 		}
 		$html .= '</ul></div>';
 
-		echo $html;
+		echo $html; // WPCS: XSS ok.
```

Comment:
Should we use `wp_kses` here for extra safety?

Reply:
I opted instead to ensure that the content of `$html` is escaped before it gets to this point, so that it's not necessary to have a complex `wp_kses` statement that becomes a maintenance headache.

Reply:
Ok. That's probably fine. I think the extra layer of safety would still be nice (especially for code that is added in the future where we may miss an escape), and I guess I don't fully understand why a `wp_kses` call would be difficult to maintain. But in any case, this looks like it's safe, so LGTM 🚀